docs/understand/sizing-guide.md

```diff
@@ -49,30 +49,36 @@ we recommend allocating at least 10 GiB - since it's a caching layer over a rela
 see [Important metrics](#important-metrics) below to understand how to size this: it should be big enough to hold all commit metadata for actively referenced commits.
 
 
-### PostgreSQL database
+### Metadata Store
 
-lakeFS uses a PostgreSQL instance to manage branch references, authentication and authorization information 
+lakeFS maintains a key-value metadata store to manage branch references, authentication and authorization information 
 and to keep track of currently uncommitted data across branches.
 
 #### Storage
-The dataset stored in PostgreSQL is relatively modest 
-as most metadata is pushed down into the object store. 
+The dataset stored in the metadata store is relatively modest as most metadata is pushed down into the object store. 
 Required storage is mostly a factor of the amount of uncommitted writes across all branches at any given point in time: 
 in the range of 150 MiB per every 100,000 uncommitted writes. 
 
 We recommend starting at 10 GiB for a production deployment, as it will likely be more than enough.
 
-#### RAM and `shared_buffers`
-Since the data size is small, it's recommended to provide enough memory to hold the vast majority of that data in RAM:
+#### RAM
```

Comment:
Would consider having just a section/header per database type, under 'Database' - each section will describe the RAM and CPU and/or other recommendations like share_bufferes.

Reply:
Each way we do it, we might encounter some issues. Maybe the solution is to create a separate page for each implementation?

Reply:
Agree, what I meant in section per database type - I would expect to have the everything related to Postgres in one place and not look for mem and cpu related to postgres in two different sections.